docs/content/docs/start/tree-start.md

```diff
@@ -238,4 +238,4 @@ useEffect(() => {
 
 Note that any `Revertible`s obtained should be disposed of by the app author in order to free up the resources that are required to revert an edit.
 
-See [undo redo support](../data-structures/tree) for more information.
+See [this blog post](https://devblogs.microsoft.com/microsoft365dev/fluid-framework-undo-redo-and-transactions-in-sharedtree/) or [undo redo support](../data-structures/tree) for more information.
```

Comment:
can we not make the second link more specific?

Reply:
unfortunately, linking to sections is broken :(
exactly why I want to split that page into more pages